modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/problem/Problem.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.vulcan.problem;
+
+import java.util.Locale;
+
+/**
+ * @author Alejandro Tardín
+ */
+public interface Problem {
+
+	public String getDetail(Locale locale);
+
+	public Status getStatus(Locale locale);
+
+	public String getTitle(Locale locale);
+
+	public String getType();
+
+	public static enum Status {
+
+		BAD_REQUEST(400, "Bad Request");
```

Comment:
Why are we inserting the 400 http code here? If we want to decouple, maybe we should get rid of it, at least in here

Reply:
This is what I wanted to review with you. I agree.

Reply:
I would say, let's get rid of this in here and move it to an independant class inside the jaxrs package

Reply:
Ok I'll simplify this as discussed.

Reply:
Done!